htmresearch/algorithms/column_pooler.py

```diff
@@ -543,7 +543,8 @@ def _rightVecSumAtNZGtThreshold_sparse(sparseMatrix,
   an eventual C implementation would.
   """
   denseArray = numpy.zeros(sparseMatrix.nCols(), dtype=GetNTAReal())
-  denseArray[sparseBinaryArray] = 1
+  if len(sparseBinaryArray) > 0:
```

Comment:
Instead of doing the check, you could just do

~~~python
denseArray[numpy.asarray(sparseBinaryArray)] = 1
~~~

This has the additional benefit of not copying the array when it's already a numpy array.